src/views/Dashboard.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <v-row>
+    <v-col cols="3">
+
+    </v-col>
+    <v-col cols="6">
+      <DataListContainer />
+    </v-col>
+    <v-col cols="3">
+
+    </v-col>
+  </v-row>
+</template>
```

Comment:
As we will be using similar structure of `v-row` and `v-col`, extract this into a separate component and add slots for left, right and middle panes.

Reply:
Yeah sure